source/frontend/src/features/mapSideBar/lease/detail/LeaseTabsContainer.tsx

```diff
@@ -201,14 +208,42 @@ export const LeaseTabsContainer: React.FC<ILeaseTabsContainerProps> = ({
     });
   }
 
+  if (
+    lease?.id &&
+    (lease.paymentReceivableType.id === ApiGen_CodeTypes_LeasePaymentReceivableTypes.PYBLBCTFA ||
+      lease.paymentReceivableType.id === ApiGen_CodeTypes_LeasePaymentReceivableTypes.PYBLMOTI) &&
+    hasClaim(Claims.COMPENSATION_REQUISITION_VIEW)
+  ) {
+    tabViews.push({
+      content: (
+        <CompensationListContainer
+          fileType={ApiGen_CodeTypes_FileTypes.Lease}
+          file={lease}
+          View={CompensationListView}
+        />
+      ),
+      key: LeaseFileTabNames.compensation,
+      name: 'Compensation',
+    });
+  }
+
   const defaultTab = LeaseFileTabNames.fileDetails;
 
+  const onSetActiveTab = (tab: LeaseFileTabNames) => {
+    const previousTab = activeTab;
+    if (previousTab === LeaseFileTabNames.compensation) {
+      const backUrl = location.pathname.split('/compensation-requisition')[0];
+      history.push(backUrl);
```

Comment:
what is this for?

Reply:
closes the comp req view panel when changing tabs.

Reply:
Seems like this may not be working as intended, see this comment:

When I leave the compensation tab while viewing a compensation the viewed compensation does not close.

Reply:
I've tried a couple of times and it works, are you taking additional steps?